benches/shrink_wrap_bench.rs

```diff
@@ -0,0 +1,57 @@
+use hurban_selector::geometry;
+use hurban_selector::operations::shrink_wrap::{ShrinkWrapOp, ShrinkWrapParams};
+
+use criterion::{black_box, criterion_group, criterion_main, BenchmarkId, Criterion};
+
+fn bench_shrink_wrap(c: &mut Criterion) {
+    let mut group = c.benchmark_group("shrink_wrap");
+    let geometry = geometry::uv_sphere([0.0, 0.0, 0.0], 1.0, 3, 3);
+
+    for density in (5u32..41u32).step_by(5) {
+        group.bench_with_input(
+            BenchmarkId::new("Batch", density.pow(2)),
+            &density,
+            |b, density| {
+                b.iter(|| {
+                    let mut op = ShrinkWrapOp::new(ShrinkWrapParams {
+                        geometry: geometry.clone(),
+                        sphere_density: black_box(*density),
+                        step: 0,
+                    });
+
+                    let mut res = None;
+                    while let Some(value) = op.next_value() {
+                        res = Some(value);
```

Comment:
Ooops, I totally forgot to perform the geometry conversion here. Interesting how much overhead there was even if there wasn't supposed to be any

Reply:
Added one more commit to measure both overheads.